src/kernels/jupyter/jupyterUtils.ts

```diff
@@ -56,7 +56,30 @@ export async function handleCertsError(
     );
     if (value === enableOption) {
         sendTelemetryEvent(Telemetry.SelfCertsMessageEnabled);
-        void config.updateSetting('allowUnauthorizedRemoteConnection', true, undefined, ConfigurationTarget.Workspace);
+        await config.updateSetting('allowUnauthorizedRemoteConnection', true, undefined, ConfigurationTarget.Workspace);
```

Comment:
This missing await was what was causing all of my test failures on CI (but not on my local machine). The test was running so fast that this setting wouldn't update by the time we tried to connect and the connection would randomly fail in other spots.